app/models/enclosure_url_builder.rb

```diff
@@ -38,9 +38,9 @@ def base_enclosure_url(podcast, episode, feed)
   end
 
   def podcast_episode_expansions(podcast, episode, feed)
-    media = episode.contents.first
+    media = episode.uncut || episode.contents.first
 
-    original_url = media.try(:original_url) || ""
+    original_url = media.try(:original_url) || "media"
```

Comment:
Makes more sense to prefer the Uncut filename, over the Contents?  Also, when you replace your `episode.uncut`, that's going to soft-delete all the previous `episode.contents`.  And we'd end up with a blank filename here, which doesn't make a great enclosure url.

Left the default of `"media"` here just in case.  But in theory, the UI shouldn't allow you to save a published episode without replacing the old file.